mne_bids/read.py

```diff
@@ -583,9 +587,8 @@ def get_head_mri_trans(bids_path):
 
     # Get MEG landmarks from the raw file
     _, ext = _parse_ext(bids_fname)
-    extra_params = None
-    if ext == '.fif':
-        extra_params = dict(allow_maxshield=True)
+    if extra_params is None:
+        extra_params = dict()
```

Comment:
ok but now I need to pass allow_maxshield explicitely with fif files. Can you add it extra_params automatically for fif files?

Reply:
But this is [more consistent with `read_raw_bids`](https://github.com/mne-tools/mne-bids/blob/master/mne_bids/read.py#L442)

Reply:
Yes but actually this is something I was thinking we should change anyway. WDYT? I wanted to propose this for a while now and always forgot. I think it doesn't make sense to have this restriction.

Reply:
you mean the defaults should be changed in MNE or MNE-BIDS?

Reply:
In MNE-BIDS

Reply:
MNE docstring says:

> Data recorded with MaxShield should generally not be loaded directly, but should first be processed using SSS/tSSS to remove the compensation signals that may also affect brain activity.

so perhaps it makes sense to have `read_raw_bids` with `None` and this function with `allow_maxshield=True` since we don't load the raw data. Let me know if you agree

Reply:
Definitely add it for this function, and maybe we should rethink / discuss read_raw_bids another time :)

Reply:
done

Reply:
read_raw_bids is different as here we only care about the coordinates in the header. Not the actual data. It's way it was done like this. allow_maxshield has not effect on the output of this function and therefore it should jsut work out of the box.